charts/falco-exporter/values.yaml

```diff
@@ -175,8 +175,10 @@ serviceMonitor:
 grafanaDashboard:
   # -- enabled specifies whether the dashboard should be deployed.
   enabled: false
-  # -- folder where the dashboard is stored by grafana.
+  # -- create and set folder_annotation to specify where the dashboard is stored in grafana.
```

Comment:
Comments should start with the variable's name.

Reply:
Should be good now, also applied this to `folderAnnotation` comment